apps/arweave/src/ar.erl

```diff
@@ -269,7 +269,13 @@ show_help() ->
 					"set coordinated_mining, cm_api_secret, and cm_exit_peer."},
 			{"cm_exit_peer (IP:port)", "The peer to send mining solutions to in the "
 					"coordinated mining mode. You need to also set coordinated_mining, "
-					"cm_api_secret, and cm_peer."}
+					"cm_api_secret, and cm_peer."},
+			{"pool_server", "Configure the node as a pool server. The pool node may not "
+					"participate in the coordinated mining."},
+			{"pool_client", "Configure the node as a pool client. The node may be an "
+					"exit peer in the coordinated mining setup or a standalone node."},
+			{"pool_api_key", "API key for the requests to pool proxy."},
+			{"pool_host", "The pool proxy address"}
```

Comment:
What do you think about `pool_proxy` instead of `pool_host`?

Reply:
Works for me! Mostly I was just looking to use one name - and I agree, `pool_host` sounds like the better of the two!